app/frontend/components/shared/select/selectors/sites-select.tsx

```diff
@@ -131,30 +131,33 @@ export const SitesSelect = observer(function ({
               }}
               render={({ field: { onChange, value } }) => {
                 return (
-                  <CreatableSelect
-                    // @ts-ignore
-                    options={pidOptions}
-                    ref={pidSelectRef}
-                    value={{
-                      label: formatPidLabel(value),
-                      value: formatPidValue(value),
-                    }}
-                    onChange={(option) => {
-                      if (!option) {
-                        setValue(siteName, null)
-                      }
-                      onChange(formatPidValue(option?.value))
-                    }}
-                    onCreateOption={(inputValue: string) => {
-                      const newValue = { label: formatPidLabel(inputValue), value: formatPidValue(inputValue) }
-                      onChange(newValue.value)
-                    }}
-                    formatCreateLabel={(inputValue: string) =>
-                      t("permitApplication.usePid", { inputValue: formatPidLabel(inputValue) })
-                    }
-                    isClearable
-                    isSearchable
-                  />
+                  <Tooltip label={t("permitApplication.pidToolTip")}>
```

Comment:
Questionable use of Tooltip, conisder using a Popover component
https://v2.chakra-ui.com/docs/components/popover/usage